packages/cli/test/helpers/prepare.js

```diff
@@ -96,7 +96,15 @@ module.exports = async function prepare(session, binaryPath, tmpFixturesDir) {
       ),
     },
     'dev-fail-on-recursion-command': {
-      'package.json': '{}',
+      'package.json': JSON.stringify({
+        scripts: {
+          build: 'echo "build script"',
```

Comment:
Are you sure this is correct?

I think this will cause the build to fail with "no public directory" instead of the correct error "recursive dev command"

Reply:
I wasn't sure if I had to emit something either, but it did pass locally

Reply:
I think the point was to verify that `devCommand` gets invoked instead, since it's set as a project setting override.